.github/workflows/quicktest.yml

```diff
@@ -0,0 +1,96 @@
+name: Quicktest
+
+on:
+  push:
+    branches-ignore:
+      - master
+  # Allow manually triggering the workflow.
+  workflow_dispatch:
+
+jobs:
+  #### QUICK TEST STAGE ####
+  # Runs the tests against select PHP versions for pushes to arbitrary branches.
+  quicktest:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php: ['5.5', '7.2', 'latest']
+
+    name: "Quick Test: PHP ${{ matrix.php }}"
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Install PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          coverage: none
+
+      # Install dependencies and handle caching in one go.
+      # @link https://github.com/marketplace/actions/install-composer-dependencies
+      - name: Install Composer dependencies - normal
+        if: ${{ matrix.php < 8.0 }}
```

Comment:
I assume this condition evaluates to `false` on `'latest'`?

Reply:
Actually, let me stabilize those conditions a little more by using `startsWith()`

Reply:
(but yes, those did work otherwise the QuickTest workflow would already have been failing)